updates for v1.4.11 (#219)
ava-labs/avalanche-docs: build/release-notes/avalanchego.md

* `network-compression-enabled`

### Prometheus Metrics
Many Prometheus metrics were renamed, and many histograms were replaced with 2 gauges.
See [here](https://github.com/ava-labs/avalanche-docs/tree/master/dashboards) for updated Grafana Dashboards.

This release also adds helper methods to the `utils/metric` package.

### RocksDB
RocksDB is no longer built by default when running the build script, and it is not included in publicly released binaries. To build AvalancheGo with RocksDB, run `export ROCKSDBALLOWED=1` in your terminal and then `scripts/build.sh`. You must do this before you can use `--db-type=rocksdb`. 

The RocksDB database now places/looks for its files in a subdirectory `rocksdb`.
Note that if you previously ran with RocksDB, you'll need to move the existing files.

### Message Compression
Nodes now compress some P2P messages.
If a peer is version >= v1.4.11, Put, Push Query, Peer List and Multiput messages sent to the peer are compressed using gzip before being sent over the network.
This reduces AvalancheGo's bandwidth usage.

### Inbound Connection Throttling
Refactored inbound connection rate-limiting and enable it by default.

### General Improvements

> * Refactor and improve performance of iteration over a database served by gRPC to a plugin.

Reviewer: ```suggestion
* Refactored and improved performance of iteration over a database served by gRPC to a plugin.
```

* On Linux, clean up the C-Chain if AvalancheGo dies ungracefully

> * Refactor P2P message definitions and move them from the `network` package.

Reviewer: ```suggestion
* Refactored P2P message definitions and move them from the `network` package.
```

> * Add VM aliases to the HTTP API server

Reviewer: ```suggestion
* Added VM aliases to the HTTP API server
```

> * Replace `1024` with `units.KiB`, etc.

Reviewer: ```suggestion
* Replaced `1024` with `units.KiB`, etc.
```

> * Improve partition tolerance by processing chits in order of the creation of the corresponding queries.

Reviewer: ```suggestion
* Improved partition tolerance by processing chits in order of the creation of the corresponding queries.
```

> ### Fuji IPs

Reviewer: all these hashes will create separate items under the table of contents as that in v1.4.5. 
<img width="1524" alt="Screen Shot 2021-07-28 at 12 33 35 PM" src="https://user-images.githubusercontent.com/12171681/127377393-17edb5e6-8ddf-4292-8ad6-ed2637b07c8a.png">

> Update the bootstrap IPs for the Fuji Testnet.

Reviewer: ```suggestion
Updated the bootstrap IPs for the Fuji Testnet.
```

> 

Reviewer: Should we use past tense in `General Improvements` and `Fuji IPs`?
